ruby/extractor/src/diagnostics.rs

```diff
@@ -199,9 +199,25 @@ impl DiagnosticLoggers {
     }
 }
 
+fn longest_backtick_sequence_length(text: &str) -> usize {
```

Comment:
I can't help but feel that what this function is called is not in line with what it actually calculates. Perhaps `backtick_count` is a better name for what it does?

Reply:
Yes, good  point. The function is not right.